prow/repoowners/repoowners.go

```diff
@@ -131,44 +131,14 @@ type RepoOwners struct {
 	log *logrus.Entry
 }
 
-// LoadRepoAliases returns an up-to-date RepoAliases struct for the specified repo.
-// If the repo does not have an aliases file then an empty alias map is returned with no error.
-// Note: The returned RepoAliases should be treated as read only.
-func (c *Client) LoadRepoAliases(org, repo string) (RepoAliases, error) {
```

Comment:
I don't think we should delete this. It isn't in use currently, but will be useful when we want to check membership of OWNERS aliases as an alternative to Github teams.

Reply:
It's strange to expose public methods that are not used and you could simply revert the extra commit if needed in the future but I reverted mine and added support for non-default branches.